tests/helpers/database.ts

```diff
@@ -0,0 +1,16 @@
+import { beforeEach, afterEach } from 'vitest';
+import { getPrisma } from '../../src/database';
+
+export function cleanDatabase() {
+  const clean = async () => {
+    const prisma = getPrisma();
+    await prisma.coreUnit.deleteMany();
+    await prisma.user.deleteMany();
+  };
+  beforeEach(async () => {
+    await clean();
+  });
+  afterEach(async () => {
+    await clean();
+  });
```

Comment:
Nit: Do you actually need to do that before AND after. Let's remove unnecessary operations to speed up tests in the future